pkg/resources/shared_database_acceptance_test.go

```diff
@@ -0,0 +1,175 @@
+package resources_test
+
+import (
```

Comment:
Added one with multiple wrong values, could also add a one with invalid identifiers, but the validation for them is kind of "not yet there" (because we are skipping the validation for AccountObjectIdentifiers and ignoring external identifiers).